docs/upgrade_to_tails_2x.rst

```diff
@@ -0,0 +1,138 @@
+
+Insert your Transfer Device into a free USB port, and mount it by navigating to Places ▸ Computer, and clicking on the encrypted disk. You will be prompted to enter its passphrase.
+
+Open a Nautilus window with admin priviledges by navigating to Applications ▸ System Tools ▸ Root Terminal. At the terminal prompt, simply type `nautilus`.
+
+Type ``ctrl`` + ``l`` and navigate to `/live/persistence/TailsData_unlocked`.
+
+|Navigate to TailsData_unlocked|
+
+Open a new tab (``ctrl`` + ``t``) and navigate to your Transfer Device. Drag and drop the backup data from your Transfer Device onto the TailsData_unlocked tab.
+
+When copying a folder, select the *Apply this action to all files* option and click *Merge* to apply to all subfolders. Then you might have to select again the *Apply this action to all files* option and click *Replace* to apply to all files.
+
+In a root terminal, or as sudo, execute the following command to fix the ownership of your personal files:
+
+::
+
+	find /live/persistence/TailsData_unlocked/ -uid 1000 -exec chown -R 1000:1000 '{}' \;
+
+Reboot your primary Tails USB and verify *everything* is still there and accessible to you, including:
+
+-	KeePassX Database
+-	PGP keys
+
+.. note:: If you are restoring a Secure Viewing Station Tails USB, you may skip to Step 7. 
+	However, if you're restoring an Admin or Journalist Tails USB, continue to Step 6.
+
+6. Reinstall SecureDrop
+-----------------------
+
+Continue checking your persistent files for the following critical assets:
+
+-	Bookmarks in Tor Browser
+-	SecureDrop files, especially the ``torrc_additions`` file in ``~/Persistent/.securedrop``.
+-	If you're an admin, also be sure the files in ``~/Persistent/securedrop/install_files/`` are available.
+
+Shut down your Tails USB on the airgapped station and move to the computer you normally use to check for submissions. At this stage, all data has been migrated and it's safe to use this Tails USB on a networked computer.
+
+Boot up Tails once again with persistence and admin privileges.
+
+.. warning:: Copy ``~/Persistent/.securedrop/torrc_additions`` to a place like your desktop.
+	You'll need these old values for the following step.
+
+Re-install Securedrop with ``cd ~/Persistent/securedrop/tails_files && sudo ./install.sh``. Once completed, test your access to the Document Interface, and, if you're a Secure Drop admin, test your ssh connection to the application and monitor servers.
+
+If you did this correctly, move on to Step 7.
+
+7. Reformat the Transfer Device
+-------------------------------
+
+Repeat step 1 to restore a USB stick to a pristine state. While it probably isn't necessary to physically destroy a Transfer Device (because LUKS-encrypted data is very hard to forensically recover), you could smash the device with a hammer until the chips containing its flash memory are broken up into pieces before disposal.
```

Comment:
I would definitely recommend a couple of passes of `/dev/urandom` over the drive though. Is there a GUI tool in Tails that can do this? If not, this guide is aimed for admins so I think it would be ok to tell them how to do it on the command line.